custom_components/ims/sensor.py

```diff
@@ -398,7 +401,7 @@ def _update_from_latest_data(self) -> None:
                 self._attr_native_value = data.current_weather.rain_chance
 
             case sensor_keys.TYPE_FORECAST_TIME:
-                self._attr_native_value = data.current_weather.forecast_time.astimezone(timezone('Asia/Jerusalem'))
+                self._attr_native_value = data.current_weather.forecast_time.astimezone()
```

Comment:
Hey, does one need to pass the 'timezone' variable here? Not a python developer so just asking just in case.

Reply:
The time format in the date string doesn't specify it explicitly, but it's IST. Without this line, the time would be considered GMT which will cause 2 or 3 hr offset